src/components/button/templates/component/style.js

```diff
@@ -132,181 +128,195 @@ export let componentStyle = `
         }
 
         .paypal-button .paypal-button-content {
-            height: 30px;
-            max-height: 30px;
-            border-radius: 15px;
-            font-size: 12px;
+            height: 32px;
+            border-radius: 16px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-pp {
-            height: 25px;
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
+            font-size: 12px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-paypal {
-            height: 24px;
+        .paypal-button.paypal-branding-unbranded .paypal-button-content  {
+            font-size: 15px;
         }
 
-        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-paypal,
-        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-venmo {
-            height: 19px;
+        .paypal-button .paypal-button-content .logo.logo-pp {
+            height: 23px;
         }
 
+        .paypal-button .paypal-button-content .logo.logo-paypal,
+        .paypal-button .paypal-button-content .logo.logo-venmo,
         .paypal-button .paypal-button-content .logo.logo-credit {
-            height: 23px;
+            height: 22px;
         }
 
-        .paypal-button .paypal-button-content::before {
-            padding: 2px;
-            top: -2px;
-            left: -2px;
-            border-radius: 17px;
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-pp {
+            height: 21px;
         }
 
-        .paypal-button .paypal-button-tag-content {
-            font-size: 10px;
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-paypal,
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-venmo,
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-credit {
+            height: 20px;
         }
 
-        .paypal-button.paypal-shape-rect .paypal-button-content,
-        .paypal-button.paypal-shape-rect .paypal-button-content::before {
+        .paypal-button.paypal-shape-rect .paypal-button-content {
             border-radius: 6px;
         }
     }
 
-
     /* Large */
 
     @media only screen and (min-width : 300px) {
+
         body {
             height: 60px;
-
         }
+
         body.fundingicons {
             height: 85px;
         }
-    }
-
-    @media only screen and (min-width : 300px) and (min-height: 60px) {
 
         .paypal-button {
             max-width: 500px;
         }
 
         .paypal-button .paypal-button-content {
-            height: 40px;
-            max-height: 40px;
-            border-radius: 20px;
-            font-size: 14px;
+            height: 42px;
+            border-radius: 21px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-pp {
-            height: 30px;
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
```

Comment:
` .paypal-button .paypal-button-tag-content {}`
Class name in `template.js` is paypal-button-tag-content for the tag line